src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbvariantcontenteditors.directive.js

```diff
@@ -101,13 +103,14 @@
                 //now re-sync any other editor content (i.e. if split view is open)
                 for (var s = 1; s < vm.editors.length; s++) {
                     //get the variant from the scope model
-                    var variant = _.find(vm.content.variants, function (v) {
-                        return (!v.language || v.language.culture === vm.editors[s].content.language.culture) && v.segment === vm.editors[s].content.segment;
-                    });
+                    var variant = vm.content.variants.find(v =>
+                        (!v.language || v.language.culture === vm.editors[s].content.language.culture) && v.segment === vm.editors[s].content.segment);
+
                     vm.editors[s].content = variant;
                 }
             }
 
+            eventsService.emit('editors.content.cultureChanged', activeVariant.language.culture);
```

Comment:
@nul800sebastiaan @nathanwoulfe with latest changes in `v8/contrib` I get an console error on e.g. HomePage node, because the document type doesn't vary by culture and `language` therefore is null.

![image](https://user-images.githubusercontent.com/2919859/88378074-94046980-cda0-11ea-9b9c-8ff994f4e4de.png)

![image](https://user-images.githubusercontent.com/2919859/88378037-82bb5d00-cda0-11ea-86c3-b2da89e6da6c.png)

Reply:
Of course. Maybe just using active variant.language instead - that way the entire language object is emitted, or it's null.

Reply:
Always wondered too why invariant content didn't populate the default language...

Reply:
Yes, it might be better with the entire language object instead and could be useful if the language object later is extended with more properties.

Reply:
#8474 fixes this